### PR TITLE
new-localStorage-connection

### PR DESCRIPTION
**Pull request by ChristophObst**

### CODE REVIEW

**Comment by ChristophObst** [comment]
trick: set intervalIterating in useeffects from 100 -> 1000 to cheat at the speed

File: pages/_app.js [hunk 1]
```diff
@@ -121,6 +91,19 @@ export default function App({ Component, pageProps }) {
     return () => clearInterval(interval);
   }, [isActive3]);
 
+  useEffect(() => {
+    setTrys(trys);
+    if (checkFruits) {
+      tryText;
+      save();
+
+      setTrys(1);
```
Comment (rabe200) [inline pages/_app.js:100]: this is confusing. shouldn't the tries be set to 0 after winning? or is this a cheating slots machine?

File: pages/_app.js [hunk 2]
```diff
@@ -121,6 +91,19 @@ export default function App({ Component, pageProps }) {
...
+    }
+    if (!checkFruits && !isActive3 && !isActive1 && !isActive2) {
+      setTrys(trys + 1);
+    }
+  }, [isActive3 || isActive2 || isActive1]);
```
Comment (rabe200) [inline pages/_app.js:105]: missing dependecies:
[checkFruits, isActive1, isActive2, isActive3, save, tryText, trys]);

Comment (rabe200) [reply pages/_app.js:105]: problem: you are calling setFunction inside useEffect. this causes infinite loop

**Review by rabe200 — CHANGES_REQUESTED** [review]
see comments

File: pages/_app.js (hunk 1 above)
Comment (ChristophObst) [reply pages/_app.js:100]: if you win, they tries will be saved in localStorage (records) "save", then as next render-step the tries will be reset to 1 (setTrys(1), maybe i will change the name to "current try"

**Pull request merged**